echoshader/new_version/echogram.py

```diff
@@ -183,56 +224,66 @@ def echogram_single_frequency(
             **kwargs: Additional keyword arguments.
 
         Returns:
-            holoviews.Image: The generated echogram visualization.
+            holoviews.DynamicMap: The generated echogram visualization with bounds.
         """
         if channel is not None:
             self.channel_select.value = channel
 
         if cmap is not None:
             self.color_map.value = cmap
 
-        list_camp = convert_string_to_list(self.color_map.value)
-        self.gram_opts["Image"]["cmap"] = (
-            list_camp if list_camp is not False else self.color_map.value
-        )
-
-        if vmin is not None and vmax is not None:
-            self.Sv_range_slider.start = vmin
-            self.Sv_range_slider.end = vmax
-            self.Sv_range_slider.value = (vmin, vmax)
+        self.gram_opts["Image"]["cmap"] = self.color_map.value
 
-        elif vmin is not None:
-            self.Sv_range_slider.start = vmin
-            old_vmax = self.Sv_range_slider.value[1]
-            self.Sv_range_slider.value = (vmin, old_vmax)
+        old_vmin = self.Sv_range_slider.value[0]
+        old_vmax = self.Sv_range_slider.value[1]
```

Comment:
minor suggestion on variable names for clarity
```suggestion
        data_vmin = self.Sv_range_slider.value[0]
        data_vmax = self.Sv_range_slider.value[1]
```